.gitignore

```diff
@@ -5,3 +5,5 @@
 *.o
 dustmite
 /cov/
+.dub
```

Comment:
I think only the directory in the root should be ignored, i.e. `/.dub` or `/.dub/`.

Reply:
Done.